application/src/main/kotlin/net/corda/v5/application/marshalling/json/JsonWriter.kt

```diff
@@ -63,12 +63,10 @@ interface JsonWriter {
      * Writes a string value.
      *
      * @param text The string to write.
-     * @param offset The offset into the text to begin writing from.
-     * @param len The length of data to write.
      *
      * @throws IOException if there is a writing error.
      */
```

Comment:
This method and the next were not supported in their original form by Jackson when I tried to plumb it in.